CONTRIBUTING.md

```diff
@@ -35,10 +35,11 @@ This SDK is linted with `Prettier` and `ESlint`.
 
 ### How to Release
 
-1. Draft New Release on GitHub.com
-2. Set tag to v0.0.0 following [Semver 2.0](https://semver.org/)
-3. Set title to "v0.0.0 Major Feature Name"
-4. Set contents to follow [KeepAChangeLog.com 1.0](https://keepachangelog.com/en/1.0.0/), but limited to just the new release information
+1. Commit version update in package.json following [Semver 2.0](https://semver.org/)
+2. Draft New Release on GitHub.com
+3. Set tag to 0.0.0 (matching what is in the package.json)
```

Comment:
Previously we have been prefixing our tags with `v`. Do we need/want to stop?

Also I was confused by set tag to 0.0.0 as I took it literally. Here's a reword suggestion:
```suggestion
3. Set tag to [package.json version]
```

Reply:
The github workflow compare will fail if theres a 'v' in front, so it will be easier without v in release tag.

Reply:
Hmm... That breaks with the npm standard tagging system. I'm not sure we want to do that. Thoughts?

(npm version major defaults to tagging with a `v` prefix.)

Reply:
Yeah... i agree. I found another little action that will output the tag without a 'v' so all good.